src/Http/Controllers/SwaggerLumeController.php

```diff
@@ -20,7 +20,11 @@ public function docs($jsonFile = null)
     {
         $filePath = config('swagger-lume.paths.docs').'/'.
             (! is_null($jsonFile) ? $jsonFile : config('swagger-lume.paths.docs_json'));
-
+        
+        if (config('swagger-lume.generate_always') && ! File::exists($filePath)) {
+            Generator::generateDocs();
```

Comment:
Could wrap this with try/catch and log error? 
```suggestion
           try {               
                Generator::generateDocs();
            } catch (\Exception $e) {
                Log::error($e);

                abort(
                    404,
                    sprintf(
                        'Unable to generate documentation file to: "%s". Please make sure directory is writable. Error: %s',
                        $filePath,
                        $e->getMessage()
                    )
                );
            }
```